src/VisualStudio/Core/Def/Implementation/Workspace/SourceGeneratedFileManager.cs

```diff
@@ -104,11 +102,6 @@ public GeneratedFileDirectoryInfo(DocumentId documentId, Type generatorType)
             // The IVsRunningDocumentTable is a free-threaded VS service that allows fetching of the service and advising events
             // to be done without implicitly marshalling to the UI thread.
             _runningDocumentTable = _serviceProvider.GetService<SVsRunningDocumentTable, IVsRunningDocumentTable>();
```

Comment:
❗ This call is not allowed within a MEF constructor

Reply:
@sharwell Do you want to file a tracking issue for removing this call?